homeassistant/components/rflink.py

```diff
@@ -402,6 +406,40 @@ def async_turn_off(self, **kwargs):
         """Turn the device off."""
         return self._async_handle_command("turn_off")
 
+class CoverableRflinkDevice(RflinkCommand):
```

Comment:
I would not put this in this file as it is only used for cover's and not multiple different components like switch/light. Best to just stick it in RflinkCover class.